README.md

```
@@ -1,16 +1,22 @@
-### Hi there 👋
+# Ben Mazzarol
 
-<!--
-**bmazzarol/bmazzarol** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.
+Hi there! I'm Ben, a software engineer with a passion for building small 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)